plutus-book/doc/09-extended.adoc

```diff
@@ -62,83 +66,133 @@ sections of a Haskell program, called Plutus Tx, which are then compiled into
 Plutus Core.
 
 Scripts are a way to add smart contract functionality
-to the extended UTxO model.
+to the extended UTXO model. The code inside Plutus scripts defines the
+contract functionality.
 Many of the subsequent chapters of this book discuss building scripts
 that define specific smart contracts. In this section, however,
-we discuss in general how scripts are used in the extended UTxO model.
-For a draft of the formal specification of the extended UTxO model with
+we discuss in general how the EUTXO model enforces the functionality defined
+by the contracts in the scripts.
+For a draft of the formal specification of the extended UTXO model with
 scripts, see <<endmatter#scripts>>.
 
-Scripts, like witnesses, are another
-tool to make sure a transaction is performing only authorized actions.
-The purpose of witnessing for (non-script) spending of outputs is to verify
+The purpose of witnessing is to verify
 that the wallet spending the output has the private key associated with the address
-in the corresponding UTxO entry (i.e. the address from which it is spending the
+in the corresponding UTXO entry (i.e. the address from which it is spending the
 money) by checking that it was indeed signed (witnessed)
 by that key.
+Scripts, like witnesses, are another
+tool to ensure a transaction is spending only outputs it is authorized to.
+
+A script to which it
+is possible to pay has its own address in the UTXO, and it is the hash of this script.
+Only spending from these _script addresses_ can be controlled by the script (whose
+address this is). Let us look at a very small example that omits some of the
+details, but gives the reader an idea of the kind of thing scripts can do.
+Suppose the owner of (non-script) address `a` has promised to pay `x` Ada
+to the first person to guess a word they are thinking of.
+This is a kind of contract, and we can interpret it as a smart contract by
+considering exactly how it controls the transfer of funds. This process
+always starts with a contribution of funds to the contract.
```

Comment:
Aside, but I really don't find the allusions to a vague separate notion of "smart contract" helps much, and especially here it seems pretty vacuous.

Reply:
In fact, I think it's actively confusing every time it appears in this chapter.